x-pack/plugins/actions/server/actions_config.ts

```diff
@@ -98,6 +100,13 @@ function getProxySettingsFromConfig(config: ActionsConfig): undefined | ProxySet
   };
 }
 
+function getResponseSettingsFromConfig(config: ActionsConfig): ResponseSettings {
+  return {
+    maxResponseContentLength: config.maxResponseContentLength,
+    responseTimeout: parseDuration(config.responseTimeout),
```

Comment:
If you do use `schema.duration()` as suggested above, you can change this to:
```suggestion
    responseTimeout: config.responseTimeout.asMilliseconds(),
```